lib/doggy.rb

```diff
@@ -23,6 +23,10 @@ module Doggy
 
   extend self
 
+  def random_word
+    (0...12).map { (97 + rand(26)).chr }.join
```

Comment:
I stared at ASCII tables too long in my life, so I say "A" and then a `downcase` and thought why not just "a" or 97.